project/src/Behaviors/GSTArrive.gd

```diff
@@ -4,12 +4,17 @@ extends GSTSteeringBehavior
 # The calculation will attempt to arrive with zero remaining velocity.
 
 
+# The target whose location the agent will be steered to arrive at
 var target: GSTAgentLocation
+# The distance from the target for the agent to be considered successfully arrived
 var arrival_tolerance: float
+# The distance from the target for the agent to begin slowing down
 var deceleration_radius: float
+# A constant that represents the time it takes to change acceleration
 var time_to_reach := 0.1
 
 
+# Initializes the behavior
```

Comment:
Don't document built-in callbacks, they're not part of the API

Reply:
This is also an example of a redundant docstring: your docstring or class description shouldn't paraphrase the symbol. Here's an example of what *not* to do:

```gdscript
class_name Animation
# An animation

# Plays the animation
func play(animation: str) -> void:
```